checker/node/examples.py

```diff
@@ -354,6 +354,7 @@
     'BNU/BNU_FGS_CCNL_v1',
     'CAS/CAS_IGSNRR_PML_V2',
     'CAS/CAS_IGSNRR_PML_V2_v017',
+    'CAS/CAS_IGSNRR_PML_V2_v018',
```

Comment:
Please add the preview script instead to show a thumbnail that makes sense to you. 

Example:
https://github.com/google/earthengine-catalog/blob/main/examples/NASA/NASA_VIIRS_002_VNP09GA_preview.js

Reply:
I created a preview file to show a thumbnail, while the 'CAS/CAS_IGSNRR_PML_V2_v018' in the examples.py has been deleted